src/common/options.cc

```diff
@@ -5276,6 +5276,14 @@ static std::vector<Option> get_rbd_mirror_options() {
     Option("rbd_mirror_leader_max_acquire_attempts_before_break", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(3)
     .set_description("number of failed attempts to acquire lock after missing heartbeats before breaking lock"),
+
+    Option("rbd_mirror_image_policy_update_check_interval", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(1)
+    .set_description("interval (in seconds) to check images for mirror daemon peer updations"),
+
+    Option("rbd_mirror_image_policy_migration_threshold", Option::TYPE_INT, Option::LEVEL_ADVANCED)
```

Comment:
`throttle` doesn't look accurate to me -- how about `rbd_mirror_image_policy_migration_interval`

Reply:
I only suggested throttle since it's what prevents the image from migrating more than X times per Y. To me, an interval implies that is will migrate every X seconds.